core/src/main/scala/kafka/message/ByteBufferMessageSet.scala

```diff
@@ -462,20 +465,22 @@ class ByteBufferMessageSet(val buffer: ByteBuffer) extends MessageSet with Loggi
 
       if (!inPlaceAssignment) {
         // Cannot do in place assignment.
-        val wrapperMessageTimestamp = {
+        val (wrapperMessageTimestamp, offsetOfMaxTimestampInMessageSet) = {
```

Comment:
The name wrapperMessageTimestamp seems inaccurate since the message set is not always compressed?